### PR TITLE
feat(nx): update to Angular 8.2, DevKit 8.3, and NgRx 8.3

### PR DESCRIPTION
**Pull request by brandonroberts**
_[Please make sure you have read the submission guidelines before posting an PR](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#submit-pr)_

> _Please make sure that your commit message follows our format._

> Example: `fix(nx): must begin with lowercase`

### CODE REVIEW

**Comment by matheo** [comment]
Hi @brandonroberts 
Is there any chance to not rely on fixed versions but on patch releases?
like `~0.803.0` to take the already released `0.803.2`.
I know there were some issues with `0.800.3` but hoping that the lesson is learned,
why we cannot trust in the core packages and its bugfixes releases?
minor releases are another story, but at least we need to trust the latest bugfixes.
Thoughts? and thanks

File: packages/angular/src/schematics/application/application.spec.ts [hunk 1]
```diff
@@ -297,6 +297,21 @@ describe('app', () => {
         getFileContent(tree, 'apps/my-dir/my-app/src/app/app.component.ts')
       ).toContain('Thank you for using and showing some ♥ for Nx.');
     });
+
+    it('should update the AppComponent spec to target Nx content', async () => {
```
Comment (vsavkin) [inline packages/angular/src/schematics/application/application.spec.ts]: Why is needed after the upgrade?

File: packages/angular/src/schematics/application/application.spec.ts [hunk 2]
```diff
@@ -113,7 +113,7 @@ describe('app', () => {
       let myAppPrefix = workspaceJson.projects['my-app'].prefix;
 
       expect(myAppPrefix).toEqual('proj');
-      expect(appE2eSpec).toContain('Welcome to my-app!');
+      expect(appE2eSpec).toContain('my-app app is running!');
```
Comment (vsavkin) [inline packages/angular/src/schematics/application/application.spec.ts]: why did this changed?

File: packages/angular/src/schematics/application/application.spec.ts (hunk 1 above)
Comment (brandonroberts) [reply packages/angular/src/schematics/application/application.spec.ts]: It was due to the changes to the HTML template for generating an application using the Angular schematics. We were just using the generated spec file, but the selectors are no longer valid.

File: packages/angular/src/schematics/application/application.spec.ts (hunk 2 above)
Comment (brandonroberts) [reply packages/angular/src/schematics/application/application.spec.ts]: This should be reverted. I'll have to update the e2e spec also to replace the text

File: packages/angular/src/utils/versions.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export const nxVersion = '*';
-export const angularVersion = '^8.0.0';
-export const angularDevkitVersion = '^0.800.1';
+export const angularVersion = '^8.2.0';
+export const angularDevkitVersion = '^0.803.2';
 export const angularJsVersion = '1.6.6';
-export const ngrxVersion = '8.1.0';
+export const ngrxVersion = '8.3.0';
 export const rxjsVersion = '~6.4.0';
```
Comment (matheo) [inline packages/angular/src/utils/versions.ts]: @brandonroberts rxjs 6.5.3 is out, any reason to not get that one synced?
and thanks for this work!
P/S. I wonder if there will be a day to sync bugfix releases, as 8.0.3 and 0.803.3 is out already

**Pull request merged**